renku/core/commands/cwl_runner.py

```diff
@@ -56,7 +56,7 @@ def addLocation(d):
     runtime_context = RuntimeContext(
         kwargs={"rm_tmpdir": False, "move_outputs": "leave", "preserve_entire_environment": True}
     )
-    loading_context = LoadingContext(kwargs={"construct_tool_object": construct_tool_object})
+    loading_context = LoadingContext(kwargs={"construct_tool_object": construct_tool_object, "relax_path_checks": True})
```

Comment:
@Panaetius should this be actually hardcoded here, or rather make it default `True` in `LoadingContext`?

Reply:
This way is fine. LoadingContext is not one of our classes anyways :slightly_smiling_face: